ts/@types/react-native-barcode-builder.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'react-native-barcode-builder';
```

Comment:
Pleas add a proper definition here.

Reply:
the file is updated with https://github.com/teamdigitale/io-app/pull/1139/commits/d8937d609376870c8e8408125fa03291e1145389